docs/configuration.md

```diff
@@ -0,0 +1,39 @@
+# Configuring _improv_
+
+## Redis Configuration Options
+
+Although _improv_ works with Redis out of the box, there are a handful of optional configuration options which can be used to change the behavior of _improv_'s data store.
+Redis configuration is specified in the configuration file under the `redis_config` key.
```

Comment:
Can you add a very simple stub of a config YAML that shows what this might look like?

Something like
```
actors: ...

connections: ...

redis_config: 
  enable_saving: True
  db_filename: "my_dump_file.rdb"
```

Reply:
This looks good. Can you include (and do we allow) users to set checkpointing frequency, as we discussed on Slack? If so, let's document here.

Reply:
We no longer do checkpointing, but I went ahead and exposed the fsync schedule of the AOF, and documented it in the docs.